client/src/MakeCall/CallCard.js

```diff
@@ -504,6 +503,23 @@ export default class CallCard extends React.Component {
         this.setState({ selectedMicrophoneDeviceId: microphoneDeviceInfo.id });
     };
 
+    async handleDialPadView() {
```

Comment:
can you use useState for state management?

Reply:
Not able to use useState in class.
 https://www.geeksforgeeks.org/how-to-use-usestate-in-arrow-function-instead-of-hook/?ref=rp

Also, other variables are used in that way in this file, that's why used that approach.